app/views/application_records/show.haml

```diff
@@ -1,15 +1,16 @@
-%h1 Application Record
+%h1.h2="#{@record.position.name} Application Record"
```

Comment:
I pointed out all the other h1s on the page. He said, and I quote "In my opinion they _are_ too big, but I don't care."